### ISSUE TITLE
Please provide the support for starting and ending point while user is drawing on the screen

### ISSUE THREAD

**Issue by darshitArkay** [EPNW/painter#43]
Right now, we don't have any callback to check from where the user is started drawing on the screen. We are not getting any coordinates on the screen. Need help with this!

**Comment by KrishnaRauniyar** [comment]
Yes I also need the starting and ending coordinate on the screen. Thank you !
